### ISSUE TITLE
Failed local deployment

### ISSUE THREAD

**Issue by basimolimajeed** [Azure/azure-iot-remote-monitoring#103]
Hi , I followed the Wiki instructions for local deployment but I am getting the following error:

Get-AzureResource : {
  "Error": {
    "Code": "ResourceNotFound",
    "Message": "The Resource
'Microsoft.DocumentDb/databaseAccounts/iotsuitelocalaed50' under resource
group 'IotSuiteLocal' was not found.",
    "Target": null,
    "Details": null
  }
}
At C:\Users\Plantronics\Documents\GitHub\azure-iot-remote-monitoring\Common\Dep
loyment\DeploymentLib.ps1:172 char:12
+     return Get-AzureResource -ResourceName $resourceName -ResourceGroupName
$res ...
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
~~~
    + CategoryInfo          : CloseError: (:) [Get-AzureResource], ErrorRespon
   seMessageException
    + FullyQualifiedErrorId : ResourceNotFound,Microsoft.Azure.Commands.Resour
   ceManager.Cmdlets.Implementation.GetAzureResourceCmdlet

Get-AzureResource : {
  "Error": {
    "Code": "ResourceNotFound",
    "Message": "The Resource
'Microsoft.Eventhub/namespaces/IotSuiteLocald629a' under resource group
'IotSuiteLocal' was not found.",
    "Target": null,
    "Details": null
  }
}
At C:\Users\Plantronics\Documents\GitHub\azure-iot-remote-monitoring\Common\Dep
loyment\DeploymentLib.ps1:172 char:12
+     return Get-AzureResource -ResourceName $resourceName -ResourceGroupName
$res ...
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
~~~
    + CategoryInfo          : CloseError: (:) [Get-AzureResource], ErrorRespon
   seMessageException
    + FullyQualifiedErrorId : ResourceNotFound,Microsoft.Azure.Commands.Resour
   ceManager.Cmdlets.Implementation.GetAzureResourceCmdlet

Provisioning resources, if this is the first time, this operation can take up 10
 minutes...
New-AzureResourceGroupDeployment : Error 1:
Code=InvalidDeploymentParameterValue; Message=The value of deployment
parameter 'docDBSku' is null. Please specify the value or use the parameter
reference. See http://aka.ms/arm-deploy/#parameter-file for details.
At C:\Users\Plantronics\Documents\GitHub\azure-iot-remote-monitoring\Common\Dep
loyment\PrepareIoTSample.ps1:127 char:11
+ $result = New-AzureResourceGroupDeployment -ResourceGroupName
$resourceGroupName ...
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
~~~
    + CategoryInfo          : CloseError: (:) [New-AzureResourceGroupDeploymen
   t], ArgumentException
    + FullyQualifiedErrorId : Microsoft.Azure.Commands.Resources.NewAzureResou
   rceGroupDeploymentCommand

WARNING: The output object of this cmdlet will be modified in a future release.
WARNING: The output object of this cmdlet will be modified in a future release.
Provisioing failed
At C:\Users\Plantronics\Documents\GitHub\azure-iot-remote-monitoring\Common\Dep
loyment\PrepareIoTSample.ps1:132 char:5
+     throw "Provisioing failed"
+     ~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : OperationStopped: (Provisioing failed:String) []
   , RuntimeException
    + FullyQualifiedErrorId : Provisioing failed

**Comment by jamesweb-ms** [comment]
It looks like this a resumption of a failed deployment.  Your ResourceGroup already exists, but the resources don't.  When we added the respect SKU logic, we forgot to check for this scenario.  Switch to the jamesweb-ImportLib branch ( proposed change #102 )  and give that a shot.  It checks for existence of the objects and let me know if that fixes the issue for you.

**Comment by basimolimajeed** [comment]
Thanks for the reply. I tried the branch you mentioned and it worked fine. Thank you

**Comment by jamesweb-ms** [comment]
Fixed in PR #102
